build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,37 @@
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int-secnet1.
+OVSIntegrationBridge: {{ (.Values.secondaryNetworkOVS).ovsIntBrName | quote }}
+
+# Name of the OpenVSwitch bridge antrea-agent will create for Secondary network specific tunnel needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-inti2.
+OVSTransportBridge: {{ (.Values.secondaryNetworkOVS).ovsTransBrName | quote }}
+
+# Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+# - system
+# - netdev
+# 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+# OVS in userspace mode. Userspace mode requires the tun device driver to be available.
+OVSDatapathType: {{ (.Values.secondaryNetworkOVS).ovsDatapathType | quote }}
+
+# Name of the OVS patch port which would connected two OVS bridges per secondary inteface config.
```

Comment:
connected two OVS bridges -> connect integration and transport bridges

Maybe remove "per secondary interface config".

Reply:
Aligned and made the changes.